routes/api.php

```diff
@@ -183,6 +183,11 @@
                 Route::post('projects/{project}/fetch', 'CurrikiGo\CourseController@fetchFromMoodle');
             });
 
+            // Specific routes for Safari Montage.
+            Route::group(['prefix' => 'safarimontage'], function () {
+                Route::post('projects/{project}/playlists/{playlist}/activities/{activity}/publish', 'CurrikiGo\PublishController@activityToSafariMontage');
```

Comment:
Better avoid horizontal scroll and move long lines to new line.

Reply:
Fixed.